spec/unit/active_fedora_spec.rb

```diff
@@ -15,6 +15,24 @@
     restore_spec_configuration
   end
 
+  describe "validate Fedora URL" do 
+
+    let(:good_url) { ActiveFedora.fedora_config.credentials[:url] }
+    let(:bad_url) { good_url.gsub('/rest', '/') }
+
+    it "should connect OK" do 
+      expect(ActiveFedora::Base.logger).to_not receive(:warn)
+      ActiveFedora::Fedora.new(:url => good_url, :base_path => '/test')
+    end
+
+    it "should not connect and warn" do 
+      expect {
+        expect(ActiveFedora::Base.logger).to receive(:warn)
```

Comment:
Should this line be on the previous line, outside the `expect { ... }.to raise_error...`?